src/lib/helper/buildPaginationHelper.ts

```diff
@@ -0,0 +1,162 @@
+import { Association } from "../../types/associations";
+import { LANDING_URL, ITEMS_PER_PAGE } from "../../constants";
+import { PageItem, PaginationData } from "../../types/pagination";
+
+export const buildPaginationElement = (
+    currentPageNumber: number,
+    numOfPages: number,
+    urlPrefix: string,
+    searchQuery: string
+): PaginationData => {
+    const pagination: PaginationData = { items: [] };
+    const pageItems: PageItem[] = [];
+
+    if (numOfPages <= 1 || currentPageNumber < 1) return pagination;
+
+    // Add Previous and Next
+    if (currentPageNumber !== 1) {
```

Comment:
If currentPageNumber is 0, which is not equal to 1, the you would have the previous page as -1

Reply:
These are the numbers linked below the table, they start from number 1. 
The current page cannot be below 1 - line 14 checks this.

Reply:
Then still it would be better to have currentPageNumber > 1 instead of !== 1. It's more precise.

Reply:
now change d to currentPageNumber > 1